core/src/main/scala/org/ensime/indexer/domain.scala

```diff
@@ -188,6 +189,7 @@ final case class ArrayDescriptor(fqn: DescriptorType) extends DescriptorType {
     case a: ArrayDescriptor => a.reifier
   }
   def internalString = "[" + fqn.internalString
+  def fqnString = fqn.fqnString
```

Comment:
it's probably best we just take the long route then and call `.fqn.fqnString` instead of trying to pretend that the descriptors always have fqns.